### DISCUSSION TITLE
Import "machine" could not be resolved Pylance

### DISCUSSION THREAD

**Discussion by eismeraldo** [BradenM/micropy-cli#445] [General]
micropy, version 4.1.0
Python: version 3.9.7
VSCode version 1.77.2
Plug-ins: Since today new Python Extension Pack

I created a project with `micropy init <PROJECT NAME>` with the following templates:
     * `VSCode Settings for Autocompletion/Intellisense`
     * `Pylint MicroPython Settings`
     * `Git Ignore Template`
     * `main.py & boot.py files`
and let the stubs for esp32-micropython-1.15.0 be created.
Unfortunately I get the above error message

Does anyone have any idea what I should do to make Pylance happy

**Comment by Alain-Godo** [comment]
Did you solve the problem? I have the same problem : ( but with the last version available for all of them.

**Comment by BradenM** [comment]
@eismeraldo Apologies, I must have missed this one.
Unsure if your issue was related to #474 or not -- but regardless I suspect it should be resolved by now. Please let me know if otherwise. Thanks.
